examples/nucleo_u575zi-q/blink/main.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <modm/board.hpp>
+
+using namespace Board;
+
+// ----------------------------------------------------------------------------
+int
+main()
+{
+	Board::initialize();
+
+	MODM_LOG_DEBUG   << "debug"   << modm::endl;
+	MODM_LOG_INFO    << "info"    << modm::endl;
+	MODM_LOG_WARNING << "warning" << modm::endl;
+	MODM_LOG_ERROR   << "error"   << modm::endl;
+
+	uint32_t counter = 0;
+
+	Rcc::connect<GpioA8::Mco>();
+	Rcc::enableClockOutput(Rcc::ClockOutputSource::Pll, Rcc::ClockOutputPrescaler::Div16);
```

Comment:
PLL output is configured to output 160 MHz, but on MCO I measure something like 6.5 MHz × 16 = 105 MHz.
Selecting HSI16 for the MCO looks fine.

https://github.com/modm-io/modm/blob/974db28da8bb9d17e9982c6c81de73cede2245ee/src/modm/board/nucleo_u575zi-q/board.hpp#L93-L103

Reply:
I have no idea what is wrong with the PLL 😒

I checked the PLL config, especially `RCC->PLL1DIVR` using GDB:
```
p/x RCC->PLL1DIVR
$ = 0x0101'0227
# -> PLL1N==39, PLL1P==1, PLL1Q==1, PLL1Q==1

p/x RCC->PLL1CFGR
$ = 0x0007'020e
# -> PLL1xEN all outputs enabled, PLL1MBOOST==0 (division by 1 (bypass)),
#     PLL1M==2, PLL1RGE==0b11 (8..16MHz), PLL1SRC==0b10 (HSI16)
```
![image](https://user-images.githubusercontent.com/2820734/224558797-94039c01-7a53-466f-aa4d-546749c34321.png)

![image](https://user-images.githubusercontent.com/2820734/224559955-be9db3b4-837c-435d-a712-75ab5331a05e.png)

Switching the MCO output (`MCOSEL` bits in `RCC->CFGR1`) to `HSI16 (0b0011)` shows a perfect 16 MHz (divided by 16) signal, but `SYSCLOCK (0b0001)` or `Main PLL pll1_r_ck (0b0101)` output something around 108 MHz (divided by 16).

The [STM32U575 Errata sheet](https://www.st.com/resource/en/errata_sheet/es0499-stm32u575xx-and-stm32u585xx-device-errata-stmicroelectronics.pdf) does not mention any PLL1 malfunction.

Reply:
Found the issue: `PLL1M` in `RCC->PLL1CFGR` has to be set to `pllFactors.pllM - 1`, so `1` in this case but it was `2`.

https://github.com/modm-io/modm/blob/974db28da8bb9d17e9982c6c81de73cede2245ee/src/modm/platform/clock/stm32/rcc.cpp.in#L210